test/integration/inflation-rewards/powerInflationZeroRewards.t.sol

```diff
@@ -257,6 +256,53 @@ contract PowerInflationZeroRewards_IntegrationTest is IntegrationBaseSetup {
         assertEq(_powerToken.balanceOf(_dave), 105_000);
     }
 
+    // TODO: should be an invariant test
+    function test_powerInflation_balancesSumEqualVotingPowerSum() external {
```

Comment:
This will not be the case. The sum of all balances will always be less than or equal to the su of all voting power, and the sum of all voting power will always be less than or equal to the total supply.